samples/web/content/apprtc/constants.py

```diff
@@ -22,3 +22,30 @@
 RESPONSE_UNKNOWN_CLIENT = 'UNKNOWN_CLIENT'
 RESPONSE_DUPLICATE_CLIENT = 'DUPLICATE_CLIENT'
 RESPONSE_SUCCESS = 'SUCCESS'
+
+BIGQUERY_URL='https://www.googleapis.com/auth/bigquery'
```

Comment:
Why can't all of these be in analytics.py?

I'm worried that we will bloat this file with constants from every module, when they only make sense in the context of that particular module. I do think that LogField and EventType shouldn't be here though, since they aren't used without the analytics module also being imported anyway.